front/src/common/Selector/Selector.tsx

```diff
@@ -148,7 +156,7 @@ const Selector = <
   }>
 ) => {
   const {
-    borderClass = 'selector-pink',
+    borderClass = 'selector-blue',
```

Comment:
Can you also remove the borderClass in CurveParamSelectors l257 please ? :)

Reply:
fixed